tt_metal/hw/inc/dev_msgs.h

```diff
@@ -65,6 +65,8 @@ struct launch_msg_t {  // must be cacheline aligned
     volatile uint16_t watcher_kernel_ids[DISPATCH_CLASS_MAX_PROC];
     volatile uint16_t ncrisc_kernel_size16;  // size in 16 byte units
 
+    volatile uint16_t host_assigned_op_id;
```

Comment:
@pgkeller was changing `launch_msg_t` in his PR here: https://github.com/tenstorrent/tt-metal/pull/9781/files#diff-d606266ae23fd80f4fa4f80ea161f8b17ff937bbf94cce50cfd731755d0a491b

He suspects that moving `run` field around is causing some issues.

Reply:
Hmm interesting, I did not see any hangs on CI with the addition of the two bytes. I will coordinate before merging this in.